src/onegov/fsi/assets/js/ifs.js

```diff
@@ -0,0 +1,26 @@
+function autoResizebyID(id) {
+    var newheight;
+    var newwidth;
+
+    if (document.getElementById) {
+        newheight = document.getElementById(id).contentWindow.document.body.scrollHeight;
+        newwidth = document.getElementById(id).contentWindow.document.body.scrollWidth;
+    }
+
+    document.getElementById(id).height = (newheight) + "px";
+    document.getElementById(id).width = (newwidth) + "px";
+}
+
+function autoResize() {
+    var newheight;
+    var newwidth;
+    var frame;
+    if (document.getElementsByClassName) {
```

Comment:
Same here, this one has been supported since IE9 and we require IE11.

Reply:
oki